index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+   
+    <header id="nav-bar">
+        <a href="#"><h1>
+              Sonu
+            </h1></a>
+            <desktop id="header-desktop">
+            <h2 id="portfolio">Portoflio</h2>
+            <h3 id="about-header">About</h3>
+            <h4 id="contact-header">Contact</h4>
+            <img src="images/letter-icon.svg" id="letter-icon">
+           <a href="#"><img src="images/Union.png" id="menu"></a>
+        </desktop>
+    </header>
+    <section id="home">
+        
+        <div id="intro">
+        <h1>
+            Hey There. I'm Sonu
+        </h1>
+        <h2>
+            I’am a Software Developer
+        </h2>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don’t hestiate to contact me.
+        </p>

```

Comment:
I suggest either giving your paragraph a width  Currently your paragraph and the background image are overrunning each other for the width allowed for mobile.
<img width="414" alt="Screenshot 2022-10-29 at 16 12 44" src="https://user-images.githubusercontent.com/48546992/198833898-5eba9987-27db-4c99-8b3b-c8abaadae2ba.png">
<img width="788" alt="Screenshot 2022-10-29 at 16 21 42" src="https://user-images.githubusercontent.com/48546992/198833903-f92b236a-2e50-4d25-98fa-3dca1074d9fe.png">

See the figma design:
<img width="374" alt="Screenshot 2022-10-29 at 16 06 59" src="https://user-images.githubusercontent.com/48546992/198833934-af7872fb-78a9-48a5-9e5a-cf164910dca5.png">